zgw/object-management/src/main/kotlin/com/ritense/objectmanagement/domain/ObjectManagement.kt

```diff
@@ -27,26 +28,29 @@ import javax.persistence.Table
 data class ObjectManagement(
     @Id
     @Column(name = "id", nullable = false, updatable = false)
-    val id: UUID = UUID.randomUUID(),
+    override val id: UUID = UUID.randomUUID(),
 
     @Column(name = "title", nullable = false, unique = true)
-    val title: String,
+    override val title: String,
 
     @Column(name = "objecttypen_api_plugin_configuration_id", nullable = false)
-    val objecttypenApiPluginConfigurationId: UUID,
+    override val objecttypenApiPluginConfigurationId: UUID,
 
     @Column(name = "objecttype_id", nullable = false)
-    val objecttypeId: String,
+    override val objecttypeId: String,
+
+    @Column(name = "object_type_version")
+    override val objecttypeVersion: Int = 1,
```

Comment:
The FE has a modal to create the ObjectManagement. That modal doesn't have the `objecttypeVersion` yet. 
Please either ask Kevin to update it, or update it yourself

Reply:
Or maybe a new story for this is needed

Reply:
Working on it.